### PR TITLE
Bugfix: Use Date mock for tests

### PR DESCRIPTION
**Pull request by ryotogashi**

### CODE REVIEW

**Comment by ryotogashi** [comment]
@axross Can you check when you have time?

File: utilities/getRelativeDateString.test.ts [hunk 1]
```diff
@@ -4,6 +4,20 @@ import {
 } from "./getRelativeDateString";
 
 describe("getRelativeDateString()", () => {
+  const RealDate = Date;
```
Comment (axross) [inline utilities/getRelativeDateString.test.ts:7]: 🤔

```suggestion
  const OriginalDate = Date;
```

File: utilities/getRelativeDateString.ts [hunk 2]
```diff
@@ -1,50 +1,52 @@
...
+  const hourDiff = Math.floor(diff / (60 * 60));
   if (hourDiff) return `${hourDiff} h`;
 
-  const minuteDiff = now.getMinutes() - date.getMinutes();
+  const minuteDiff = Math.floor(diff / 60);
   if (minuteDiff) return `${minuteDiff} ${minuteDiff === 1 ? "min" : "mins"}`;
 
-  const secondDiff = now.getSeconds() - date.getSeconds();
-  return `${secondDiff} ${secondDiff === 1 ? "sec" : "secs"}`;
+  return `${diff} ${diff === 1 ? "sec" : "secs"}`;
```
Comment (axross) [inline utilities/getRelativeDateString.ts:51]: What's the purpose of these changes? 🤔 Seems not related to the change using date mock

Comment (ryotogashi) [reply utilities/getRelativeDateString.ts:51]: That is true. It isn't related to using date mock. I'll delete it and create another issue.

Comment (axross) [reply utilities/getRelativeDateString.ts:51]: Nice, thanks!
